src/base/bittorrent/session.cpp

```diff
@@ -2063,6 +2063,23 @@ bool Session::addTorrent_impl(CreateTorrentParams params, const MagnetUri &magne
             patchedFastresumeData.replace("6:pausedi0e", "6:pausedi1e");
             patchedFastresumeData.replace("12:auto_managedi0e", "12:auto_managedi1e");
 
+            // converting relative save_path to absolute
+            int start = patchedFastresumeData.indexOf("9:save_path");
+            if (start > -1) {
+                start += 11;
+                const int end = patchedFastresumeData.indexOf(":", start);
+                const int len = patchedFastresumeData.mid(start, end - start).toInt();
+                if (len > 0) {
+                    const QByteArray relativePath = patchedFastresumeData.mid(end + 1, len);
+                    const QByteArray absolutePath = Profile::instance().fromPortablePath(Utils::Fs::toUniformPath(fromLTString(relativePath))).toUtf8();
+                    if (relativePath != absolutePath) {
+                        const QByteArray replaceBefore = QByteArray("9:save_path").append(QString::number(len).toUtf8()).append(":").append(relativePath);
```

Comment:
You should use `QByteArray::number(len)` instead of `QString::number(len).toUtf8()`.

Reply:
Thanks, changed `QString::number(...).toUtf8()` to `QByteArray::number(...)`.